help/bulleted-and-numbered-lists.md

```diff
@@ -0,0 +1,11 @@
+# Bulleted and numbered lists
+
+Zulip supports Markdown formatting for bulleted and numbered lists.
+
+{!lists.md!}
```

Comment:
We might want a naming scheme more like `format_lists.md`, just to avoid potential naming collisions / improve readability in the future.